raft/raft.go

```diff
@@ -1428,14 +1428,15 @@ func (r *raft) restore(s pb.Snapshot) bool {
 	// code here and there assumes that r.id is in the progress tracker.
 	found := false
 	cs := s.Metadata.ConfState
+outer:
 	for _, set := range [][]uint64{
 		cs.Voters,
 		cs.Learners,
 	} {
 		for _, id := range set {
 			if id == r.id {
 				found = true
-				break
+				break outer
 			}
 		}
```

Comment:
avoid the jump tag. check found here for break.

Reply:
@xiang90 
Did you mean something like this ?
```
		for _, id := range set {
			if id == r.id {
				found = true
				break
			}
		}
		if found {
			break
		}
```

Reply:
yes.

Reply:
@xiang90 
Change has been made.